### ISSUE TITLE
fpm detected as down

### ISSUE THREAD

**Issue by Grummfy** [Lusitaniae/phpfpm_exporter#7]
Hello,
my fpm process is detected as down (using sockets).
the socket is well detected but _up stats = 0.

Any idea where it came from (the process is running well)

**Comment by Lusitaniae** [comment]
Hi @Grummfy 

Do you have this on your nginx config?
```

**Comment by Lusitaniae** [comment]
Did you enable the status page on fpm?

If you use 127.0.0.1:9001 instead of a socket can you read the output? (you probably also need nginx config )

**Comment by Grummfy** [comment]
no, I was thinking it was required for the non-socket version. I will try
